gdk/common/utils.py

```diff
@@ -152,6 +154,15 @@ def is_recipe_size_valid(file_path):
     return file_size <= MAX_RECIPE_FILE_SIZE_BYTES, file_size
 
 
+def generate_ignore_list_from_globs(root_directory, globs):
```

Comment:
Why is it not inside Zip? Who else is going to need this?

Reply:
Moved it to Zip file